appengine/standard/firebase/firetactoe/static/main.js

```diff
@@ -29,7 +29,9 @@
 function initGame(gameKey, me, token, channelId, initialMessage) {
   var state = {
     gameKey: gameKey,
-    me: me
+    me: me,
+    channelId: channelId,
+    initialMessage: initialMessage
```

Comment:
Is there a reason to save the initialMessage? Its contents are going to be incorporated into the state almost immediately anyway.